assets/stylesheets/bootstrap/_theme.scss

```diff
@@ -152,6 +155,7 @@
 // Inverted navbar
 .navbar-inverse {
   @include gradient-vertical($start-color: lighten($navbar-inverse-bg, 10%), $end-color: $navbar-inverse-bg);
+  @include reset-filter; // Remove gradient in IE<10 to fix bug where dropdowns don't get triggered; see https://github.com/twbs/bootstrap/issues/10257
```

Comment:
reset-filter looks 100% IE<10, are you sure we need this? At the very least remove the mention of IE9 if you're 100% sure that this is somehow required (doubtful).